app/controllers/root_controller.rb

```diff
@@ -24,14 +24,15 @@ def govuk_component_docs
   end
 
   def govuk_available_locales
-    locale_files = Rails.root.join('app', 'views', 'locales', '*.yml')
+    locale_files = Rails.root.join("config", "locales", "*.yml")
     locales = Dir[locale_files].map { |file| File.basename(file, '.yml') }
     render :json => locales
   end
 
   def govuk_locales
     return error_404 unless params[:locale].match(/^[a-z]{2}(-[a-z0-9]{2,3})?$/)
```

Comment:
I'm not familiar with the internals of the i18n for Rails so not sure, this seems like a sanity check but maybe something like https://github.com/tigrish/iso#tags would be more appropriate.